frontend/package.json

```diff
@@ -3,28 +3,23 @@
   "version": "0.1.0",
```

Comment:
I've pinned each version exactly. We've got a minimal set of direct dependencies but still better safe than investigating random errors because we pulled in a new version :)